doc/source/management/data_at_rest_encryption.rst

```diff
@@ -1,804 +1,11 @@
-  ``keyring_key_remove`` operation supported by the ``keyring_udf`` plugin).
-
-* ``vault_ca [optional]`` - this variable needs to be specified only when the
-  Vault's CA certificate is not trusted by the machine that is going to connect
-  to the Vault server. In this case this variable should point to CA
-  certificate that was used to sign Vault's certificates.
-
-.. warning::
-
-   Each ``secret_mount_point`` should be used by only one server - otherwise
-   mixing encryption keys from different servers may lead to undefined
-   behavior.
-
-An example of the configuration file looks like this: ::
-
-  vault_url = https://vault.public.com:8202
-  secret_mount_point = secret
-  token = 58a20c08-8001-fd5f-5192-7498a48eaf20
-  vault_ca = /data/keyring_vault_confs/vault_ca.crt
-
-When a key is fetched from a ``keyring`` for the first time the
-``keyring_vault`` communicates with the Vault server, and retrieves the key
-type and data. Next it queries the Vault server for the key type and data and
-caches it locally.
-
-Key deletion will permanently delete key from the in-memory hash map and the
-Vault server.
-
-.. note::
-
-  |Percona XtraBackup| currently doesn't support backup of tables encrypted
-  with :ref:`keyring_vault_plugin`.
-
-System Variables
-----------------
-
-.. variable:: keyring_vault_config
-
-  :cli: ``--keyring-vault-config``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Text
-  :default:
-
-This variable is used to define the location of the
-:ref:`keyring_vault_plugin` configuration file.
-
-.. variable:: keyring_vault_timeout
-
-  :cli: ``--keyring-vault-timeout``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Numeric
-  :default: ``15``
-
-This variable allows to set the duration in seconds for the Vault server
-connection timeout. Default value is ``15``. Allowed range is from ``1``
-second to ``86400`` seconds (24 hours). The timeout can be also completely
-disabled to wait infinite amount of time by setting this variable to ``0``.
-
-.. _data-at-rest-encryption.innodb-system-tablespace:
-
-
-
-InnoDB System Tablespace Encryption
 ================================================================================
-
-:Availabiliity: This feature is **Experimental** quality
-
-The InnoDB system tablespace is encrypted by using master key encryption. The
-server must be started with the ``--bootstrap`` option.
-
-If the variable :variable:`innodb_sys_tablespace_encrypt` is set to ON and the
-server has been started in the bootstrap mode, you may create an encrypted table
-as follows:
-
-.. code-block:: guess
-
-   mysql> CREATE TABLE ... TABLESPACE=innodb_system ENCRYPTION='Y'
-
-.. note::
-
-   You cannot encrypt existing tables in the System tablespace.
-
-It is not possible to convert the system tablespace from encrypted to
-unencrypted or vice versa. A new instance should be created and user tables must
-be transferred to the desired instance.
-
-You can encrypt the already encrypted InnoDB system tablespace (key rotation)
```

Comment:
This will re-encrypt encryption key stored in tablespace's header with new MK, but it will not re-encrypt the table itself. We might want to say that clearly here, as we want to advertise that encryption threads can do actual tablespace re-encryption.

Reply:
Good. Will add.